### ISSUE TITLE
Can't add ZemiSmart Zigbee Hub

### ISSUE THREAD

**Issue by PharmCat** [rospogrigio/localtuya#1572]
## The problem

Error: Connection to device succeeded but no datapoints found, please try again. Create a new issue and include debug logs if problem persists.

When try to add:
"model": "ZMHK-01",
 "name": "ZemiSmart Zigbee Hub",

## Environment

- Localtuya version: last
- Home Assistant Core version: 2023.9.2
- [X] Does the device work using the Home Assistant Tuya Cloud component ? 
- [] Does the device work using the Tinytuya (https://github.com/jasonacox/tinytuya) command line tool ? 
- [] Was the device working with earlier versions of localtuya ? Which one? 
- [X] Are you using the Tuya/SmartLife App in parallel ? 

## Steps to reproduce

Try to add ZemiSmart Zigbee Hub

**Comment by nuraci** [comment]
I can confirm the same problem happens to me.
Same configuration on Intel mini pc with HAOS last version,
